index.html

```diff
@@ -0,0 +1,200 @@
+    <ul class="navigation__list">
+      <li class="navigation__item"><button type="button" class="navigation__item__button">섬</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">국립공원</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">통나무집</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">기상천외한 숙소</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">해변 근처</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">초소형 주택</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">디자인</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">캠핑장</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">A자형 주택</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">호숫가</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">북극</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">멋진 수영장/button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">동굴</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">서핑</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">최고의 전망</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">복토 주택</button></li>
+      <li class="navigation__item"><button type="button" class="navigation__item__button">열대지역</button></li>
+    </ul>
+    <button type="button" class="navigation__left arrow-button" aria-label="이전 숙소 유형 더보기"></button>
+    <button type="button" class="navigation__right arrow-button" aria-label="다음 숙소 유형 더보기"></button>
+    <button type="button" class="navigation__filter">필터</button>
+  </nav>
+  <main>
+    <h3 class="sr-only">섬 유형의 숙소들</h3>
+    <section class="accommodation__container">
+      <a href="/" class="accommodation__link">
+        <article class="accommodation__item">
+          <!-- h4 변경 -->
+          <h4 class="sr-only">몰디브에 위치한 워터 방갈로</h4>
+          <div class="accommodation__item__image">
+            <!-- src, alt 변경 -->
+            <img src="/" alt="">
+            <button type="button" class="arrow-button" aria-label="다음 숙소 이미지 더보기"></button>
+          </div>
+          <h5 class="sr-only">숙소 정보</h5>
+          <!-- 숙소 정보 변경 -->
+          <ul class="accommodation__item__brief">
+            <li class="font-bold">MV, 몰디브</li>
+            <li>6,596km 거리</li>
+            <li>9월 24일~29일</li>
+            <li class="font-bold">₩781,552 /박</li>
```

Comment:
`/박` 이 부분은 폰트가 굵은 폰트가 아니어서
아래처럼 수정할 수 있을 것 같아요 !

```suggestion
            <li><span class="font-bold">₩781,552</span> /박</li>
```

Reply:
오 제가 놓친부분이네요. 아주 섬세한 커멘트 감사합니다!!(●'◡'●)